_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1492,38 +1496,24 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
 
     MFX_CHECK_WITH_ASSERT(VA_STATUS_SUCCESS == vaSts, MFX_ERR_DEVICE_FAILED);
 
-
-    m_caps.CQPSupport =
-        (attrs[idx_map[VAConfigAttribRateControl]].value & VA_RC_CQP) ? 1 : 0;
-    m_caps.CBRSupport =
-        (attrs[idx_map[VAConfigAttribRateControl]].value & VA_RC_CBR) ? 1 : 0;
-    m_caps.VBRSupport =
-        (attrs[idx_map[VAConfigAttribRateControl]].value & VA_RC_VBR) ? 1 : 0;
-    m_caps.ddi_caps.VCMBitrateControl =
-        (attrs[idx_map[VAConfigAttribRateControl]].value & VA_RC_VCM) ? 1 : 0; //Video conference mode
-    m_caps.ddi_caps.ICQBRCSupport =
-        (attrs[idx_map[VAConfigAttribRateControl]].value & VA_RC_ICQ) ? 1 : 0;
+    m_caps.CQPSupport = !!(AV(VAConfigAttribRateControl) & VA_RC_CQP);
```

Comment:
This is not "slight code refactoring", this is behavior change since we start fully ally on driver caps reporting. I can't say that this is wrong, but it worth to state this explicitly in the commit message to warn those who will step into problems and bisect to this commit. Can you, please, update commit message, please?

Reply:
Not a problem to update the commit message but I don't see new behavior.. Please give line example.

Reply:
I don't follow. Which error? Please give a link to line.